cloudcix_primitives/bridgeifns.py

```diff
@@ -0,0 +1,359 @@
+        namespace:
+            description: The VRF network namespace identifier, such as 'VRF123'.
+            type: string
+            required: true
+
+    return:
+        description: |
+            A tuple with a boolean flag indicating if the interface was successfully deleted,
+            and the output or error message.
+        type: tuple
+    """
+
+    # Define message
+    messages = {
+        1100: f'1100: Successfully removed interface {namespace}.{bridgename} inside namespace {namespace}',
+        1101: f'1101: Interface {namespace}.{bridgename} does not exist',
+
+        3121: f'3121: Failed to connect to the enabled PodNet from the config file {config_file} for payload interface_check:  ',
+        3122: f'3122: Failed to connect to the enabled PodNet from the config file {config_file} for payload interface_del:  ',
+        3123: f'3123: Failed to run interface_del payload on the enabled PodNet. Payload exited with status ',
+
+        3151: f'3151: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_check:  ',
+        3152: f'3152: Failed to connect to the disabled PodNet from the config file {config_file} for payload interface_del:  ',
+        3153: f'3153: Failed to run interface_del payload on the disabled PodNet. Payload exited with status ',
+    }
+
+    # Default config_file if it is None
+    if config_file is None:
+        config_file = '/opt/robot/config.json'
+
+    status, config_data, msg = load_pod_config(config_file)
+    if not status:
+        if config_data['raw'] is None:
+            return False, msg
+        else:
+            return False, msg + "\nJSON dump of raw configuration:\n" + json.dumps(config_data['raw'],
+                                                                               indent=2,
+                                                                               sort_keys=True)
+    enabled = config_data['processed']['enabled']
+    disabled = config_data['processed']['disabled']
+
+    def run_podnet(podnet_node, prefix, successful_payloads):
+        rcc = SSHCommsWrapper(comms_ssh, podnet_node, 'robot')
+        fmt = PodnetErrorFormatter(
+            config_file,
+            podnet_node,
+            podnet_node == enabled,
+            {'payload_message': 'stdout', 'payload_error': 'stderr'},
+            successful_payloads
+        )
+
+        payloads = {
+            'interface_check': f'sudo ip netns exec {namespace} ip link show dev {namespace}.{bridgename}',
+            'interface_del':  f'sudo ip netns exec {namespace} ip link del {namespace}.{bridgename}'
+        }
+
+        ret = rcc.run(payloads['interface_check'])
+        if ret["channel_code"] != CHANNEL_SUCCESS:
+            return False, fmt.channel_error(ret, f"{prefix+1}: " + messages[prefix+1]), fmt.successful_payloads
+        if ret["payload_code"] != SUCCESS_CODE:
+            #If the interface already does NOT exists returns info and true state
```

Comment:
Use similar logic as in the open pull request for build here e.g `delete_interface = False`